.travis.yml

```diff
@@ -7,18 +7,24 @@ language: python
 python:
 - 3.5
 - 3.6
+before_install:
+- curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+- source ~/.nvm/nvm.sh
+- nvm install 10
+- node --version
+- npm i yarn -g
 install:
 - pip install -r requirements.txt
-- npm install
+- yarn
```

Comment:
Why not npm (in other words, is yarn _opinionated_)?

Reply:
It has the saying that npm may generate a wrong dep tree. (I'm not sure about this)
and yarn seems to be faster add it will flatten the modules. It takes smaller disk space.

Reply:
Vote for Yarn. Npm generates a black hole but Yarn not.